src/test/resources/testdata/sql/seed-sar-data-1.sql

```diff
@@ -0,0 +1,830 @@
+                                     status_changed_to, submission_date)
+VALUES (41, 1, 'RESETTLEMENT_PLAN', '{
+  "assessment": [
+    {
+      "answer": {
+        "@class": "StringAnswer",
+        "answer": "NO"
+      },
+      "questionId": "HELP_TO_MANAGE_ANGER"
+    },
+    {
+      "answer": {
+        "@class": "StringAnswer",
+        "answer": "NO"
+      },
+      "questionId": "ISSUES_WITH_GAMBLING"
+    },
+    {
+      "answer": {
+        "@class": "StringAnswer",
+        "answer": "SUPPORT_NOT_REQUIRED"
+      },
+      "questionId": "SUPPORT_NEEDS_PRERELEASE"
+    },
+    {
+      "answer": {
+        "@class": "StringAnswer",
+        "answer": "hello"
+      },
+      "questionId": "CASE_NOTE_SUMMARY"
+    }
+  ]
+}', '2024-05-22 16:05:17.294788', 'Matthew Kerry', 'hello', 'MKERRY_GEN', 'ATTITUDES_THINKING_AND_BEHAVIOUR',
+        'SUBMITTED', 'SUPPORT_NOT_REQUIRED', '2024-05-22 16:06:53.590886');
+INSERT INTO resettlement_assessment (id, prisoner_id, assessment_type, assessment, created_date, created_by,
+                                     case_note_text, created_by_user_id, pathway, assessment_status,
+                                     status_changed_to, submission_date)
+VALUES (42, 1, 'RESETTLEMENT_PLAN', '{
+  "assessment": [
+    {
+      "answer": {
+        "@class": "StringAnswer",
+        "answer": "NO_ANSWER"
+      },
+      "questionId": "PARTNER_OR_SPOUSE"
+    },
+    {
+      "answer": {
+        "@class": "StringAnswer",
+        "answer": "NO"
+      },
+      "questionId": "PRIMARY_CARER_FOR_CHILDREN"
+    },
+    {
+      "answer": {
+        "@class": "StringAnswer",
+        "answer": "NO_ANSWER"
+      },
+      "questionId": "CARING_FOR_ADULT"
+    },
+    {
+      "answer": {
+        "@class": "StringAnswer",
+        "answer": "NO_ANSWER"
+      },
+      "questionId": "SUPPORT_FROM_SOCIAL_SERVICES"
+    },
+    {
+      "answer": {
+        "@class": "StringAnswer",
+        "answer": "NO_ANSWER"
+      },
+      "questionId": "FRIEND_FAMILY_COMMUNITY_SUPPORT"
+    },
+    {
+      "answer": {
+        "@class": "StringAnswer",
+        "answer": "NO_ANSWER"
+      },
+      "questionId": "INVOLVEMENT_IN_GANG_ACTIVITY"
+    },
+    {
+      "answer": {
+        "@class": "StringAnswer",
+        "answer": "NO_ANSWER"
+      },
+      "questionId": "UNDER_THREAT_OUTSIDE"
+    },
+    {
+      "answer": {
+        "@class": "StringAnswer",
+        "answer": "NO_ANSWER"
+      },
+      "questionId": "COMMUNITY_ORGANISATION_SUPPORT"
+    },
+    {
+      "answer": {
+        "@class": "StringAnswer",
+        "answer": "SUPPORT_DECLINED"
+      },
+      "questionId": "SUPPORT_NEEDS_PRERELEASE"
+    },
+    {
+      "answer": {
+        "@class": "StringAnswer",
+        "answer": "hello"
+      },
+      "questionId": "CASE_NOTE_SUMMARY"
+    }
+  ]
+}', '2024-05-22 16:05:40.499062', 'Matthew Kerry', 'hello', 'MKERRY_GEN', 'CHILDREN_FAMILIES_AND_COMMUNITY',
```

Comment:
weren't me guv

Reply:
Yeah shamelessly copied the data from another sql file